pkg/machine/fedora.go

```diff
@@ -0,0 +1,129 @@
+// +build amd64 arm64
+
+package machine
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	githubURL      = "http://github.com/fedora-cloud/docker-brew-fedora/"
```

Comment:
Should be in containers.conf

Reply:
ok will add these

Reply:
I do not think that is needed we already have image path in containers.conf, this is just for the default download path right?

Reply:
That'll probably work if we make the defaults OS-specific. Linux and Darwin will use the current default, Windows will use the other.

Reply:
This was the approach I was taking, so far it seems to work out. Although this wsl impl does ignore the hardware settings since they are dynamically acquired (you can impose global limits (shared across all linux distributions for a user), but the common case is not to). The existing qemu approach will ignore the distro URL since it uses a fedoracoreos vendor module that defines it statically. Even though not all options are supported by each backend I think that adding another block doesnt help make it cleaner to describe, but let me know if you guys prefer seperation. BTW I am trying to turn this around quickly but I have an internal conference and back to back meetings all week, so probably wont have a PR till late evening.

Reply:
`podman machine init` already has `--image-path` which can also be set in containers.conf so as long as this PR also allows to set a custom image url for windows I do not see why we need to change anything in containers.conf.